packages/@o3r/core/schematics/rule-factories/otter-environment/index.ts

```diff
@@ -112,7 +112,7 @@ export function updateOtterEnvironmentAdapter(
 
     }
     if (options.isDefaultGenerator) {
-      workspace.cli.defaultCollection = '@o3r/core';
+      workspace.cli.schematicCollections = (workspace.cli.schematicCollections || []).concat('@o3r/core');
```

Comment:
Why don't we use `registerPackageCollectionSchematics` in `@o3r/schematics`

Reply:
I think you mean `registerCollectionSchematics`